includes/wc-stripe-functions.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * WooCommerce Stripe Functions
+ *
+ * General functions available on both the front-end and admin.
+ *
+ * @package WooCommerce_Stripe\Functions
+ * @version 10.1.0
+ */
+
+if ( ! function_exists( 'str_contains' ) ) {
```

Comment:
We don't need our own polyfill for this, as a polyfill for `str_contains()` has been in WordPress since 5.9.0 as per https://github.com/WordPress/wordpress-develop/commit/3cc8f1237a737ccf895682029afacafd7305d420. Given that 5.9.0 was released in early 2022 and our minimum supported version is 6.7, I think we should rely on the WordPress core polyfill.

Reply:
Thanks for pointing that out! Removed it in https://github.com/woocommerce/woocommerce-gateway-stripe/pull/4737/commits/44602b6e1196fe8e78b38fbb336e4ae8420fcdbb